test/extended/deployments/deployments.go

```diff
@@ -1549,4 +1550,64 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			o.Expect(strings.TrimSpace(rcs.Items[0].Spec.Template.Spec.Containers[0].Image)).NotTo(o.BeEmpty())
 		})
 	})
+
+	g.Describe("running pods [Conformance]", func() {
+		dcName := "nettest"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("Terminating pod should not be considered part of the set of running pods", func() {
```

Comment:
I don't think this is a test for DC. The accounting of pods is copied over from ReplicaitionController so the test would be suited to be placed there but it looks more like a unit test